pages/views.py

```diff
@@ -98,29 +98,23 @@ def prepare_page_range(paginator, current_page):
         }
 
 
-class RobotsView(View):
+class RobotsView(CustomPageView):
 
-    # `filter` prevents query evaluation
-    objects = CustomPage.objects.filter(slug='robots')
-    template = 'robots.txt'
+    template_name = 'robots.txt'
     in_db = False
-    content_type = 'text/plain'
 
     def get(self, request, *args, **kwargs):
-        context = {
+        context = self.get_context_data()
+        if self.in_db:
```

Comment:
why we need `in_db` var?
We can suggest, that our robots always in db (it's true for every site).

If we throw away in_db, we can simplify RobotsView or rm it